Quicksilver/QuicksilverDmgVersioner.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/local/autopkg/python
+#
+# Copyright 2010 Per Olofsson
```

Comment:
Needs updating.

Reply:
I assume you mean the copyright needs attending to? Tricky as my version is just a mashup of Per's work with Greg's FoundationPlist, but I've done my best :)